pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -101,72 +103,98 @@ def test_importer_config(self):
                 self.assertEqual(importers[0][key], body['importer_' + key])
 
 
-class SyncValidFeedTestCase(utils.BaseAPITestCase):
-    """Create an RPM repository with a valid feed and sync it.
+# This class is left public for documentation purposes.
+@unittest2.skip('Base test case.')
+class SyncRepoBaseTestCase(utils.BaseAPITestCase):
```

Comment:
Would not be better to have this base class on the utils module?

This way I think the skip will not be needed and we will not require to remember that one skip is due to this.